src/Compatibility/Microsoft.DotNet.Compatibility/MSBuildCompatibilityLogger.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+using Microsoft.DotNet.ApiCompatibility.Logging;
+using Microsoft.NET.Build.Tasks;
+
+namespace Microsoft.DotNet.Compatibility
+{
+    internal class MSBuildCompatibilityLogger : CompatibilityLoggerBase
```

Comment:
Please add triple slash docs to the new type.

Reply:
NIT: inherit doc is fine for the members after you document the base, but it would be good to add a description for the class itself, so we easily understand when it is used.

Reply:
Also, one more NIT: all of these new types should be `sealed`